comp/metadata/inventoryagent/inventoryagent_test.go

```diff
@@ -209,6 +214,11 @@ func TestInitData(t *testing.T) {
 		"system_probe_max_connections_per_message":     10,
 	}
 
+	if !kernel.IsIPv6Enabled() {
+		expected["system_probe_track_tcp_6_connections"] = false
+		expected["system_probe_track_udp_6_connections"] = false
+	}
```

Comment:
Is this new ?

Reply:
Yes, the system-probe has some post-processing based on the kernel used to run the test. This is why the test was failing in some runner.